pom.xml

```diff
@@ -296,7 +296,14 @@
           <findbugsXmlOutputDirectory>${project.build.directory}/findbugs
           </findbugsXmlOutputDirectory>
           <excludeFilterFile>${project.basedir}/findbugs-exclude.xml</excludeFilterFile>
-        </configuration>
+  </configuration>
+        <dependencies>
+	  <dependency>
+	    <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.9.4</version>
```

Comment:
This is ant findbugs fix from:
https://github.com/spotify/docker-client/pull/171
This affected me